### ISSUE TITLE
`dist` folder is missing from both repo and releases

### ISSUE THREAD

**Issue by BSarmady** [gridstack/gridstack.js#2665]
## Subject of the issue
`dist` folder is missing from both repo and releases
Which neither of demos will work and Getting started section in [website](https://gridstackjs.com/) is invalid too.

## Your environment
N/A

## Steps to reproduce
N/A

## Expected behavior
`dist` folder need to be included.

**Comment by adumesny** [comment]
by design as /dist require a build step. demos are live on website https://gridstackjs.com/

**Comment by BSarmady** [comment]
@adumesny I have seen your website and then I came here.

The **Getting Started** section does not mention the build step. 

I'm fullstack developer but not a typescript one and never needed to use either node or npm in my web frontend projects. 
I don't think `<script src="node_modules/gridstack/dist/gridstack-all.js"></script>` emphasizing on  `node_modules` is the way front-end people address JavaScript in their applications, well unless you are not supporting pure javascript, html, css web applications. I always put the complete library in either assets/plugin/ or assets/vendor/

the minified script that I can download from [your website](https://gridstackjs.com/node_modules/gridstack/dist/es5/gridstack-all.js) seems to me compiled for ES3 since it does not support any of changes made in ES6 and after.

anyway, I forked your project and will try and compile it to JS to create a valid javascript dist folder which includes a full and a minified version with map files.

Thanks again
